src/styles/components/_tabs.scss

```diff
@@ -1,8 +1,8 @@
 nav.tabs {
-  --flow-spacing: var(--gcds-spacing-600);
+  margin-block-start: var(--gcds-spacing-600);
```

Comment:
This is not equivalent. We shouldn't add margin into our components. They might end up in a place where we don't want margins.

Reply:
But all the flow class does is add `margin-block-start` so how is it different to add it directly to the tabs?

Reply:
the flow class does add margin-block-start to the first children. Therefore, the margin is in between direct children of the flow parent. 

Here's a bit more info on that technique if you're curious!
- https://andy-bell.co.uk/my-favourite-3-lines-of-css/
- https://every-layout.dev/layouts/stack/

Reply:
Yes, I'm very aware of that technique, I just misread the one line yesterday evening when rewriting.

Reply:
It's not that I don't want to, I just think it would be really cool and a big selling point for the utility framework, if we can say at the end, that we almost styled the entire site with the UF minus a little bit of custom code so I wanna try using it wherever possible :)